pyscf/tools/trexio.py

```diff
@@ -260,8 +260,6 @@ def det_to_trexio(mcscf, norb, nelec, filename, backend='h5'):
 
     det_list = []
     for a, b, coeff in zip(occsa, occsb, ci_values):
-      occsa_upshifted = [orb + 1 for orb in a]  
-      occsb_upshifted = [orb + 1 for orb in b]
       det_tmp     = []
       det_tmp    += trexio_det.to_determinant_list(occsa_upshifted, int64_num)
```

Comment:
@NastaMauger you forgot to rename `occsa_upshifted` to `occsa` here (and same for `occsb`).

Reply:
My bad ! It is done now